docs/installation.rst

```diff
@@ -2,62 +2,53 @@
 Installation
 ************
 
-STIPS is a simulation tool that depends on other modules such as PSF and exposure time calculators.
-These underlying submodules need to be installed for STIPS to function properly along with their supporting datasets.
-There are multiple options for installation and they are listed in this section along with instructions.
+STIPS is a simulation tool that depends on other modules such as PSF and exposure time calculators.  These underlying submodules need to be 
+installed for STIPS to function properly along with their supporting datasets.  There are multiple options for installation and they are listed 
+in this section along with instructions.
 
 STIPS Requirements
 ##################
 
-* `Pandeia`: Exposure time calculator.
-* `WebbPSF`: James Webb and Nancy Grace Roman PSF calculator.
+* `Pandeia>=1.7`: Exposure time calculator.
+* `WebbPSF>=1.0.0`: Nancy Grace Roman PSF calculator. STIPS also requires that `poppy`, a support package used by `webbpsf`, have version `>=1.0.3`
 * `astropy`: STIPS uses astropy in order to:
 
 	- Read and write FITS files.
 	- Read and write ASCII tables (specifically in the IPAC format).
 	- Generate Sersic profile models (if any are in the generated scene).
 
 * `esutil`: Used for retrieving data from sqlite databases in the form of numpy arrays.
+  .. warning::
+     `esutil` is not installed by the stips setup.py because its pip installation has 
+     errors. After installing STIPS, you must run `pip install esutil --no-cache-dir` to
+     have `esutil` available.
+  .. note::
+     `esutil` is only needed if you are using `stips.star_generator`
```

Comment:
```suggestion

.. warning::
   `esutil` is not installed by the stips setup.py because its pip installation has
   errors. After installing STIPS, you must run `pip install esutil --no-cache-dir` to
   have `esutil` available.

.. note::
   `esutil` is only needed if you are using `stips.star_generator`

```
- Tabbing the warning prevented it from rendering properly.
- Warnings and notes require line breaks both above and below.